addon/styles/_frost-link.scss

```diff
@@ -214,10 +215,69 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
     }
 
     .svg {
-      margin-left: 10px;
+      &:not(.action) {
```

Comment:
Having action override a default would be preferable, since there could be multiple cases and it's easier to list what is included in one place rather than what is excluded across multiple places